### ISSUE TITLE
Possible improvement and correction for Player Condition wiki page.

### ISSUE THREAD

**Issue by 1010todd** [endless-sky/endless-sky#9799]
### Location

https://github.com/endless-sky/endless-sky/wiki/Player-Conditions

### Describe the issue

1. At `"roll: <input>"`, usage of `[0, input)` notation is a little too advanced. It's a topic barely (if at all) covered by high school math which is rarely used and most probably forget about it as soon as they left high school. Assuming the reader even completed high school, evidently we have people who's probably not in high school yet but also engage in ES content creation already so we probably should avoid using it or provide explanation in plain English. 

2. Now that both `"cargo attractiveness"` and `"armament deterrence"` become autocondition and not saved to the savefile anymore and thus basically make it impossible to get the actual value for these conditions unless you look inside the source code to figure out how to calculate it, which defeats the point of a wiki.
Probably the same for `"pirate attraction"` raid percentage which is mentioned on the wiki.

3. The description for `"cargo attractiveness"` seems to be straight up wrong.
```
"cargo attractiveness" is how attractive the size of your cargo hold(s) is to pirates. 
Lots of small ships are more attractive than one large one. 
Values for single human ships range from -2 for ships with no cargo to 8 for Bulk Freighters.
```
When I looked in to the source code the calculation appears to be `max(0., .4 * sqrt(attributes.Get("cargo space")) - 1.8)`. I'm bad at math but I think 0 is more than -2.

### Expected content

1. Provide explanation in plain English for `[0, input)`.

2. Provide the equation for `"cargo attractiveness"`, `"armament deterrence"`, and `"pirate attraction"` raid percentage.

3. Correct from "... -2 for ships with no cargo" to "... 0 for ships with no cargo".

**Comment by warp-core** [comment]
I've updated the descriptions for the roll and cargo attractiveness conditions:
https://github.com/endless-sky/endless-sky/wiki/Player-Conditions/_compare/fc2de5185b4980cb3b38c5b44d4e3ddce4583470
https://github.com/endless-sky/endless-sky/wiki/Player-Conditions/_compare/d32b6aec58130ff5cd2876fb9087b56c85ca0f67

**Comment by TheGiraffe3** [comment]
@1010todd if you know what it should be, you can now make a PR to fix/change it.

**Comment by petervdmeer** [comment]
Closing; I just looked at the wiki, and all 3 issues described in this ticket appear to be addressed; The roll input does have some plain English text around it, the `cargo attractiveness`, `armament deterrence` and `pirate attraction` seem to have quite complete descriptions, and the wrong description appears to be fixed.
